client/my-sites/customer-home/cards/tasks/renew/index.js

```diff
@@ -37,63 +37,56 @@ const Renew = ( { card, moment, purchases, site, siteSlug } ) => {
 		: translate( '%(planName)s plan expiring soon', { args: { planName } } );
 	let description;
 	let actionText;
-	switch ( true ) {
```

Comment:
Was this broken in some way?

Reply:
That was part of an old commit that I reverted later. I replaced that with a more simple if/else approach.

I initially used a [`switch (true)` approach](https://medium.com/trabe/using-switch-true-in-javascript-986e8ad8ae4f) since that allows to build if/else comparisons more efficiently/readable in some cases, but I realized later it was not worth here.

Reply:
I discovered switching on true by accident myself and have been in love with it ever since. <3